doc/cephfs/disaster-recovery-experts.rst

```diff
@@ -213,118 +213,142 @@ Using an alternate metadata pool for recovery
 
 .. warning::
 
-   There has not been extensive testing of this procedure. It should be
-   undertaken with great care.
+   This procedure has not been extensively tested. It should be undertaken only
+   with great care.
 
-If an existing file system is damaged and inoperative, it is possible to create
-a fresh metadata pool and attempt to reconstruct the file system metadata into
-this new pool, leaving the old metadata in place. This could be used to make a
-safer attempt at recovery since the existing metadata pool would not be
-modified.
+If an existing file system is damaged and inoperative, then it is possible to
```

Comment:
You'd think obvious from context, but to be sure I'd go with "existing CephFS file system"